h5bp/Front-end-Developer-Interview-Questions
# add a website

New website is made w/ https://11ty.io and runs via `npm`. Main commands:

* `npm start` run this at localhost:9090
* `npm run build` generate the dist folder `_site`
* `npm run deploy` push the `_site` folder to `gh-pages`

Test Plan:

1. Check out this branch
2. `npm i`
3. `npm start`
4. Check that all files are generated in http://localhost:9090

Preview:

<img width="1727" alt="website" src="https://user-images.githubusercontent.com/4672033/49017477-1671fb80-f170-11e8-8947-f1fecc81be3c.png">

We will need to update the readme/contributing docs to point to the new docs path which was moved to website's `src/`. Also, need to add instructions on how to build the website.
